lib/loggly/common.js

```diff
@@ -258,7 +258,7 @@ common.loggly = function () {
 // This function will store logs into buffer
 //
   function storeLogs(logs) {
-    if (!logs.length) return;
+    if (!logs.length || !bufferOptions) return;
```

Comment:
@mchaudhary @mostlyjason The same condition lies here too. I put this check to store the logs only when the **bufferOptions** object is set and containing the value for **retriesInMilliSeconds** and **size** otherwise this code line will throw an error.